block-explorer/v3.yml

```diff
@@ -2211,7 +2213,7 @@ components:
           type: string
           nullable: true
       example:
-        identifier: DAG2V3L7efRpk16UGcPHLcqLJthzWxpMoPR3vToD
+        id: DAG2V3L7efRpk16UGcPHLcqLJthzWxpMoPR3vToD
```

Comment:
excuse the ignorance if there is some other motivation but isn't this an Address whereas an Id is a public key?

Reply:
it was decided on a previous meeting that it should be changed from `identifier` to `id` even though the `id` originally meant to be the public key
cc: @AlexBrandes